packages/tooling/openmrs/src/commands/assemble.ts

```diff
@@ -17,6 +17,7 @@ export interface AssembleArgs {
   target: string;
   mode: string;
   config: Array<string>;
+  myConfigs: Array<string>;
```

Comment:
Better naming suggestion:

```suggestion
  configFiles: Array<string>;
```

Reply:
Okay will work on this